File: new_json_combinator/clean_files.js
const fs = require('fs');
const path = require('path');
const dir = 'd:/WORK/CLIENTS/extract/prompt-db-local/new_json_combinator';
const files = fs.readdirSync(dir).filter(f => f.endsWith('.tsx') || f.endsWith('.json'));
files.forEach(file => {
  const filepath = path.join(dir, file);
  const content = fs.readFileSync(filepath, 'utf8');
  const cleaned = content.replace(/^\s*\d+→\d+\s*/gm, '');
  fs.writeFileSync(filepath, cleaned, 'utf8');
  console.log('Cleaned:', file);
});
console.log('Done cleaning all files!');

File: new_json_combinator/clean_files.py
import os
import re

dir = 'd:/WORK/CLIENTS/extract/prompt-db-local/new_json_combinator'

for filename in os.listdir(dir):
    if filename.endswith('.tsx') or filename.endswith('.json'):
        filepath = os.path.join(dir, filename)
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        # Remove line number prefix like "     1→1" or "1→1" at start of lines
        cleaned = re.sub(r'^\s*\d+→\d+\s*', '', content, flags=re.MULTILINE)
        with open(filepath, 'w', encoding='utf-8') as f:
            f.write(cleaned)
        print(f'Cleaned: {filename}')
